examples/quickstarts/helidon-quickstart-graalvm/pom.xml

```diff
@@ -0,0 +1,310 @@
+                            <configuration>
+                                <executable>java</executable>
+                                <arguments>
+                                    <argument>-classpath</argument>
+                                    <classpath />
+                                    <argument>${main.class}</argument>
+                                </arguments>
+                            </configuration>
+                        </plugin>
+                    </plugins>
+                </pluginManagement>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>exec-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>build-native-image</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>${native.image}</executable>
+                                    <arguments>
+                                        <argument>-H:Name=${basedir}/target/${native.name}</argument>
+                                        <argument>-H:IncludeResources=${native.resources}</argument>
+                                        <!--                                <argument>&#45;&#45;debug-attach</argument>-->
+                                        <argument>-H:+ReportExceptionStackTraces</argument>
+                                        <!-- this is a workaround for issue with server, when a new library with a Java service
+                                        is added, the server cannot find its implementation class-->
+                                        <argument>--no-server</argument>
+                                        <argument>-classpath</argument>
+                                        <classpath/>
+                                        <argument>${main.class}</argument>
+                                    </arguments>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>run-native-image</id>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <executable>${basedir}/target/${native.name}</executable>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>native-image-docker</id>
+            <build>
+                <pluginManagement>
```

Comment:
Move pluginManagement in the default profile, no need to isolate these unless it conflicts with another profile.

Reply:
OK, done.